war/src/main/less/base/spacing.less

```diff
@@ -0,0 +1,45 @@
+@unit: 0.4rem;
+
+each(range(0, 5), {
```

Comment:
I'm going to suggest you go with the same sizing as established by bootstrap margin and padding utilities. That way those familiar with it will find it easier.